src/bin/find.js

```diff
@@ -38,3 +56,9 @@ Object.keys(options).forEach(function findRules(option) {
 if (processExitCode) {
   process.exit(processExitCode)
 }
+
+function getOutputRuleCellMapper(width) {
```

Comment:
May be refactoring the function (duplicate, exists in `src/bin/diff.js` and `src/bin/find.js`) out of these two files, when tackling #7